src/Tests/HelixTasks/AssemblyScheduler.cs

```diff
@@ -0,0 +1,361 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace Microsoft.DotNet.SdkCustomHelix.Sdk
```

Comment:
Did you have to write this code, or is it boilerplate?  Does every repo using Helix need to duplicate this code?

Reply:
This @jaredpar 's code https://github.com/jaredpar/arcade/blob/rolex/Rolex/AssemblyScheduler.cs

It should go into arcade one day.

> Does every repo using Helix need to duplicate this code?

By default arcade helix sdk will only distribute by test dll. Using this, you can potentially distribute one workload for each test class. There is a knob you can change. Unfortunately, due to Helix VM scale algorithm, it turned out slower, if it is too granular

Reply:
Any chance we could put it in arcade now?

Reply:
I'll send an email tomorrow. I'll see if I can check in that code to arcade and publish a nuget library. We don't completely use Arcade xunit integration since TestFramework.csproj has a commandline. So I don't won't design how it integrate into xunit command line today.

Reply:
I'd prefer to let this work go forward now and then switch it to use the arcade implementation when it's available. I'm still 2-3 weeks away from getting my code into Arcade. Don't want to block progress here on my messed up schedule.

Reply:
@nguerrera @dsplaisted  so we can merge it like that?